tests/regression-sprint-1.coffee

```diff
@@ -108,6 +133,25 @@ describe 'regression-sprint-1', ->
       $("#footer h6").html().assert_Contains('TEAM Mentor v')
       done()
 
+  it 'User Sign Up Fail (different passwords)',(done)->
```

Comment:
Is this test related to an issue? if not it should be at https://github.com/TeamMentor/TM_4_0_QA/blob/master/tests/user-account.coffee 

in fact I just looked at that file, and it seems that this test is already there https://github.com/TeamMentor/TM_4_0_QA/blob/master/tests/user-account.coffee#L59